test/python/WMCore_t/Storage_t/Backends_t/GFAL2Impl_t.py

```diff
@@ -112,15 +112,17 @@ def getStageOutCommandResult(self, copyCommandDict, createRemoveFileCommandResul
             EXIT_STATUS=$?
             echo "gfal-copy exit status: $EXIT_STATUS"
             if [[ $EXIT_STATUS != 0 ]]; then
-               echo "ERROR: gfal-copy exited with $EXIT_STATUS"
-               echo "Cleaning up failed file:"
-               %s
+                echo "ERROR: gfal-copy exited with $EXIT_STATUS"
+                echo "Cleaning up failed file:"
+                {remove_command}
             fi
             exit $EXIT_STATUS
-            """ % createRemoveFileCommandResult
-
+            """.format(
```

Comment:
to avoid pycodestyle complaints, please move it all under the same line (it is not too long)